administrator/components/com_users/Table/NoteTable.php

```diff
@@ -103,21 +107,20 @@ public function publish($pks = null, $state = 1, $userId = 0)
 			// Nothing to set publishing state on, return false.
 			else
 			{
-				$this->setError(\JText::_('JLIB_DATABASE_ERROR_NO_ROWS_SELECTED'));
```

Comment:
We need to throw an exception here instead with the same text and catch it in a good place